lib/ts/recipe/webauthn/index.ts

```diff
@@ -311,7 +357,7 @@ export default class RecipeWrapper {
      *
      * @returns `{ status: "OK", ...}` if successful along a description of the user details (id, etc.) and email
      */
-    static registerAndSignUp(input: { email: string; options?: RecipeFunctionOptions; userContext: any }): Promise<
+    static registerUserWithSignUp(input: { email: string; options?: RecipeFunctionOptions; userContext: any }): Promise<
```

Comment:
I think this name sounds a bit confusing. What would be the difference between register and signUp ?

Reply:
Agree with the changes, I have made them.